### PR TITLE
add support for ARM Linux to install script

### PR DESCRIPTION
**Pull request by cobalt2727**
both 32 bit and 64 bit environments should be working properly with this

### CODE REVIEW

File: redream [hunk 1]
```diff
@@ -166,6 +191,18 @@ if [[ "$PLATFORM" == "mac" ]]; then
 	create_macos_application
 elif [[ "$PLATFORM" == "linux" ]]; then
 	create_linux_desktop_launcher
+	if [[ "$REDREAM_ARCH" == "universal-raspberry" ]]; then
+		if [[ $launch_args == *"system-mesa"* ]]; then
+			#exec "$REDREAM_DIR/redream"
+			echo "Using the system library over the bundled one..."
+			cp $REDREAM_DIR/releases/v*/redream.$REDREAM_ARM_TYPE.elf $REDREAM_DIR/redream
```
Comment (andrewmackrodt) [inline redream]: To copy the correct file from the extracted release you can use `$RELEASE_PATH`. Quoting the paths will also ensure compatibility for installations where either path contains spaces, e.g.

```sh
cp "$RELEASE_PATH/redream.$REDREAM_ARM_TYPE.elf" "$REDREAM_DIR/redream"
```

**Comment by cobalt2727** [comment]
come to think of it, wouldn't it make more sense to install the binary to `/usr/local/bin/redream`?

**Comment by cobalt2727** [comment]
@andrewmackrodt what changes need to be made to get this merged?

**Comment by andrewmackrodt** [comment]
@cobalt2727 sorry I didn't mean to close the PR, it happened when I deleted the `windows` branch. I've merged it as part of https://github.com/andrewmackrodt/redream-launcher/commit/36916e8122c749bbe0c3d87ffcdde1b1845323d8. I've made some modifications, e.g. redream pi elf files (now?) have the filename _redream.**aarch**{32,64}.elf_.

I've also created a shell script to export the correct mesa path (I think). I don't have a pi to test with so apologies if I've broken anything.

Thanks for the contribution!

**Comment by cobalt2727** [comment]
Thank you! Just to be safe I'll give it a test with my ARM boards whenever I get the time to and report back whether or not it still works

**Pull request closed without merging**